### DISCUSSION TITLE
Support for Hebrew/Arabic

### DISCUSSION THREAD

**Discussion by ColmShannon** [moment/luxon#1614] [Q&A]
Hi,

I've encountered some issues using toLocaleString and am wondering if I'm missing something. 

`DateTime.fromISO(new Date().toISOString()).setLocale(localeLang).toLocaleString();`

Using the above snippet today (29th of March) if I pass localeLang 'ar' I get '29‏/3‏/2024'

I believe the correct format should be dd/MM/yyyy. Is there something I'm overlooking?

Thanks,
Colm

**Comment by diesieben07** [comment]
Firstly, your way of constructing the `DateTime` is a little weird, why not use `DateTime.now()`?

Luxon uses `Intl.DateTimeFormat` for formatting, which produces exactly the same result:
```js
const dtf = new Intl.DateTimeFormat('ar', {year: "numeric", month: "numeric", day: "numeric"});
console.log(dtf.format(new Date()));
```

I am not sure if this is correct, but if you think it isn't, you'll have to take this up with whatever underlying platform (browser, Node JS, etc.) you are using).

**Reply by ColmShannon** [reply]
@diesieben07 thanks, I noticed the issue existed even when using Intl shortly after posting. 

Ah update to as-EG seems to have done the trick: 

![image](https://github.com/moment/luxon/assets/74541166/72f42407-ec13-4d9d-ad07-a123354b10a2)
